### DISCUSSION TITLE
How can I play wav file in Python API?

### DISCUSSION THREAD

**Discussion by trappedinspacetime** [coqui-ai/TTS#2262] [General Q&A]
In Readme  examples regarding _Python API_ 

     wav = tts.tts("This is a test! This is also a test!!", speaker=tts.speakers[0], language=tts.languages[0])
     
How can I play that `wav` in python?

**Comment by lexkoro** [comment]
One of this should work.

https://pypi.org/project/PyAudio/

https://pypi.org/project/playsound/

**Reply by trappedinspacetime** [reply]
I am new to Python. I don't know how to apply it.  I tried a few things but I failed. I need an example.
